config/locales/de.yml

```diff
@@ -109,3 +111,14 @@ de:
                 - Bewertung
                 - Beschreibung
                 - Zeit
+  surveys:
+    index:
+      tooltips:
+        already_submitted: "Du hast diese Umfrage bereits ausgefüllt. Danke!"
+      admin_badges:
+        submission:
+          zero: "Noch keine Einreichungen"
+          one: "Eine Einreichung"
+          other: "%{count} Einreichungen"
```

Comment:
This seems to not exist in any other translation

Reply:
Those are dynamic translations of Rails i18n framework for pluralization

Reply:
They gettin used in the bubbles of the list (see image attached to PR)

Reply:
Ah so only admin, okay then it's fine